docker/db-migration/migrations/025-add-description-prop.yaml

```diff
@@ -0,0 +1,17 @@
+databaseChangeLog:
+  - changeSet:
+      id: tag
+      author: dstudeny
+      changes:
+        - tagDatabase:
+            tag: 025-add-description-prop
```

Comment:
There will be a conflict with [025-optimise-switch-connected-device-class.yaml](https://github.com/telstra/open-kilda/pull/5108/files)

Reply:
Updated.